Makefile

```diff
@@ -625,7 +626,8 @@ rest_vm_tests    = $(filter-out $(passing_vm_tests), $(all_vm_tests))
 
 test-all-vm: $(all_vm_tests:=.run)
 test-rest-vm: $(rest_vm_tests:=.run)
-test-vm: $(passing_vm_tests:=.run)
+test-vm: poetry build-kevm build-llvm
+	$(MAKE) -C kevm-pyk/ test-integration TEST_ARGS+='-k test_vm -n8'
 
 all_bchain_tests     = $(wildcard tests/ethereum-tests/BlockchainTests/GeneralStateTests/*/*.json)                            \
                        $(wildcard tests/ethereum-tests/LegacyTests/Constantinople/BlockchainTests/GeneralStateTests/*/*.json)
```

Comment:
Do we need to keep all the logic which enumerates the tests in the Makefile anymore?

Reply:
Good point. A few targets, namely `test-all-vm`, `test-rest-vm`, `test-all-bchain`, `test-rest-bchain`, were not covered by the Python test harness. I added test functions accordingly and cleaned up the `Makefile`.